WindowManager/app/src/main/java/com/example/windowmanagersample/embedding/SplitActivityDetail.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.windowmanagersample.embedding
+
+import android.content.Intent
+import android.graphics.Color
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.example.windowmanagersample.R
+
+open class SplitActivityDetail : AppCompatActivity() {
```

Comment:
I'm seeing some strange behavior for the `SplitActivityList`/`SplitActivityDetail` when rotating the device. It looks like once I launch a second `SplitActivityDetail`, rotating the device again causes the app to exit. Is that expected behavior here?

Reply:
Can you check again with this version?
I cannot reproduce this.

Reply:
Here's the behavior I'm seeing:

https://user-images.githubusercontent.com/4217560/142693733-5c98394f-b265-4a72-b1e3-a4316069e0a3.mp4

I think what is causing this is `finishPrimaryWithSecondary="true"`, since switching that to `false` changed the behavior to what I was expecting, so this might be normal.

Reply:
I cannot reproduce this issue.